core/src/main/java/com/scalar/db/config/ConfigUtils.java

```diff
@@ -54,6 +54,18 @@ public static int getInt(Properties properties, String name, int defaultValue) {
     }
   }
 
+  public static Integer getInt(Properties properties, String name, Integer defaultValue) {
+    String value = trimAndReplace(properties.getProperty(name));
+    if (Strings.isNullOrEmpty(value)) {
+      return defaultValue;
+    }
+    try {
+      return Integer.parseInt(value);
+    } catch (NumberFormatException ignored) {
+      throw new IllegalArgumentException("the specified value of '" + name + "' is not a number");
```

Comment:
How about showing the actual `value` in the log message? Just containing the cause exception is enough?

Reply:
I think showing the actual `value` in the log message is better. I will do that in this RP. Thank you!

Reply:
Fixed it in b090379c3e52e4f39f9b05fb5ea9d83eaa73d41d. 

@komamitsu Can you please take a look again when you have time? Thanks.